config/default.yml

```diff
@@ -3853,6 +3853,12 @@ Style/TrailingCommaInArrayLiteral:
     - consistent_comma
     - no_comma
 
+Style/TrailingCommaInBlockArgs:
+  Description: 'Checks for useless trailing commas in block arguments.'
+  Enabled: false
```

Comment:
According to the new [versioning strategy](https://docs.rubocop.org/en/latest/versioning/), I think this new cop should have been set with `Enable: pending`?